packages/mask/src/components/shared/ApplicationBoard.tsx

```diff
@@ -289,6 +289,7 @@ export function ApplicationBoard({ secondEntries, secondEntryChainTabs }: MaskAp
                         ChainId.xDai,
                         ChainId.Celo,
                         ChainId.Fantom,
+                        ChainId.Avalanche,
```

Comment:
https://github.com/DimensionDev/Maskbook/pull/5398

Reply:
@septs I'll wait for the merge then update

Reply:
done here 863cc8a